LOLInfoApp/LOLInfoApp/Commons/Constants/StringConstants.swift

```diff
@@ -10,4 +10,5 @@ import Foundation
 enum StringConstants {
     static let separator: String = "/"
     static let lineBreak: String = "\n"
+    static let comma: String = ","
```

Comment:
더블 쿼우팅이 있기 때문에, 타입을 명시하지 않아도 추론에 문제가 없어 보입니다. 

```suggestion
enum StringConstants { 
    static let separator = "/"
    static let lineBreak = "\n"
    static let comma = ","
}
```

Reply:
타입추론이 될때는 적극 활용하라는 피드백을 한번 주셨던 내용인데 간과했습니다. 앞으로 주의하도록 하겠습니다. 반영했습니다!